dev-java/objenesis/objenesis-1.3.ebuild

```diff
@@ -8,7 +8,7 @@ JAVA_PKG_IUSE="source doc"
 inherit java-pkg-2 java-pkg-simple
 
 DESCRIPTION="A small Java library with one purpose: To instantiate a new object of a class"
-HOMEPAGE="https://${PN}.googlecode.com/"
+HOMEPAGE="https://objenesis.googlecode.com/"
```

Comment:
fine, still useless though

Reply:
I think the point was to make CI happy now